lib/proxy/http_download.rb

```diff
@@ -14,11 +14,15 @@ def initialize(src, dst, read_timeout = nil, connect_timeout = nil, dns_timeout
       dns_timeout ||= DEFAULT_CONNECT_TIMEOUT
       connect_timeout ||= DEFAULT_DNS_TIMEOUT
 
-      args = [wget, "--connect-timeout=#{connect_timeout}",
+      args = ["--connect-timeout=#{connect_timeout}",
               "--dns-timeout=#{dns_timeout}",
               "--read-timeout=#{read_timeout}",
-              "--tries=3", "-nv", "-c", src.to_s, "-O", dst.to_s]
-      args << "--no-check-certificate" unless verify_server_cert
+              "--tries=3",
+              "--timestamping", # turn on timestamping to prevent redownloads
+              "--no-if-modified-since", # but use HTTP HEAD instead If-Modified-Since
```

Comment:
--no-if-modified-since does not work on EL7.
```
# wget --no-if-modified-since
wget: unrecognized option '--no-if-modified-since'
Usage: wget [OPTION]... [URL]...

Try `wget --help' for more options.
```